UI/src/app/components/area/area.component.html

```diff
@@ -0,0 +1,4 @@
+<h4 class="title">
```

Comment:
Component CSS styles are encapsulated into the component's view and don't affect the rest of the application if ```encapsulation ``` hasn't been set in the special way. So there is no need to create a bunch of multiple css classes to style components. Ge4t rid of unnecessary classes in your code to make it cleaner